### ISSUE TITLE
access denied Node v12.18.1

### ISSUE THREAD

**Issue by Cartman0** [coreybutler/nvm-windows#552]
Make sure you have reviewed the [common issues](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues) and existing issues before submitting a new issue.

If this is a question about how to use NVM4W, please use [stackoverflow](http://stackoverflow.com/questions/tagged/nvm-windows) instead.

If this is an issue regarding antivirus, make sure you search the existing issues first.

### My Environment

- [ ] Windows 7 or below (not truly supported due to EOL - see wiki for details)
- [ ] Windows 8
- [ ] Windows 8.1
- [○] Windows 10
- [ ] Windows 10 IoT Core
- [ ] Windows Server 2012
- [ ] Windows Server 2012 R2
- [ ] Windows Server 2016

- [○] My Windows installation is non-English.

### I'm using NVM4W version:

Run `nvm version` if you don't know.

1.1.7

### I have already...

- [○] read the README to be aware of npm gotchas & antivirus issues.
- [○] reviewed the [wiki](https://github.com/coreybutler/nvm-windows/wiki) to make sure my issue hasn't already been resolved.
- [○] verified I'm using an account with administrative privileges.
- [ ] searched the issues (open and closed) to make sure this isn't a duplicate.
- [ ] made sure this isn't a question about how to use NVM for Windows, since [gitter](https://gitter.im/coreybutler/nvm-windows?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) is used for questions and comments.

### My issue is related to (check only those which apply):

- [ ] settings.txt
- [ ] proxy support (Have you tried version 1.1.0+?)
- [ ] 32 or 64 bit support (Have you tried version 1.1.3+?)
- [ ] Character escaping (Have you tried version 1.1.6+?)
- [ ] A standard shell environment (terminal/powershell)
- [ ] A non-standard shell environment (Cmder, Hyper, Cygwin, git)

### Steps to reproduce the problem:

maybe same this issue
https://stackoverflow.com/questions/50563188/access-denied-issue-with-nvm-in-windows-10?noredirect=1&lq=1 .

```
> nvm install 12.18.1 64
> nvm use 12.18.1
Now using node v12.18.1 (64-bit)
> nvm list
12.18.1
```

it does't mark current version.

```
node -v
access denied
```

**Comment by coreybutler** [comment]
Have you tried this? https://stackoverflow.com/questions/50563188/access-denied-issue-with-nvm-in-windows-10

`access denied` means the user account does not have privileges to run the executable.

**Comment by Cartman0** [comment]
hi.
i cant solve this such as the comment  in it.

**Comment by coreybutler** [comment]
I cannot recreate the problem.

![image](https://user-images.githubusercontent.com/770982/87210803-80ef9500-c2dc-11ea-9fe9-dd8b7a551914.png)

**Comment by Cartman0** [comment]
ok.
can you also work when install nvm outside c:users/(username)/ dir (ex. c:nvm ?

**Comment by coreybutler** [comment]
Yes, it works in other directories, including `c:\nvm`. Do not set it to `C:\Program Files`, because that is a protected directory (it requires additional permissions).

**Comment by Cartman0** [comment]
um...
node v12.18.2 can't start in my env.

i changed install location nvm to `c:\nvm`, don't change symlink.

setting.txt:
```
root: C:\nvm
path: C:\Program Files\nodejs
```

As follow,  when `nvm use`, can't mark version, then `node -v`, access denied.

![nvm](https://user-images.githubusercontent.com/10578703/87217634-55c18000-c386-11ea-80df-f2d7c0db4777.PNG)

**Comment by Cartman0** [comment]
i noticed one, but i don't know it cause.

DL link "https://nodejs.org/dist/v12.18.2/win-x64//node.exe" is correct, but occured follow error.

```
Error while downloading https://nodejs.org/dist/v12.18.2/win-x64//node.exe - read tcp 192.168.1.7:49487->104.20.22.46:443: wsarecv: An existing connection was forcibly closed by the remote host.
```

**Comment by coreybutler** [comment]
Are you behind a firewall? Clearly it didn't download completely. That error is being thrown directly from the underlying network connection.

**Comment by Cartman0** [comment]
@coreybutler 
it is same to off firewall of windows defender..
firewall warning dialog also doesn't be popuped.

**Comment by coreybutler** [comment]
@Cartman0 - Are you behind a proxy? Can you download the installation file manually?

**Comment by ItsMartin030** [comment]
I'm experiencing the same issue right now. Already disabled my UAC screen when running as admin and it still shows me "Access denied"..
Reinstalled it many times and runned it as admin, didn't work.

**Comment by mpiber** [comment]
> I'm experiencing the same issue right now. Already disabled my UAC screen when running as admin and it still shows me "Access denied"..
> Reinstalled it many times and runned it as admin, didn't work.

@ItsMartin030 I was having the same problem, the solution for me was to uninstall the current version and install [this](https://github.com/coreybutler/nvm-windows/releases/download/1.1.7/nvm-setup.zip) as an administrator.

**Comment by cope** [comment]
I have the same problem, and uninstalling and installing as admin did not help :(

**Comment by mpiber** [comment]
> I have the same problem, and uninstalling and installing as admin did not help :(
> 
> ![image](https://user-images.githubusercontent.com/112515/134920087-cc8e68b5-2edb-4d58-8693-b3f66ea1e000.png)

@cope uninstall version 1.1.8 and install version [1.1.7](https://github.com/coreybutler/nvm-windows/releases/download/1.1.7/nvm-setup.zip) as an administrator, for me it was the solution.

**Comment by cope** [comment]
@mpiber oh, that worked

so then the problem is in 1.1.8 ?

**Comment by Rupam-Shil** [comment]
Yes, I guess the problem is with the version, I uninstalled version 1.1.8 and installed 1.1.7 and it started working

**Comment by coreybutler** [comment]
See https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#permissions-exit-1-exit-5-access-denied

If you feel an issue is unresolved, please open a new issue instead of commenting on closed issues. I don't make a habit of reviewing closed issues.

**Comment by eldoctorgonzo** [comment]
Hey guys, it's a permission problem. 
In Windows make sure your run the windows shell or CLI as Administrator.
Then u can use:
**nvm use latest**
to change to the last Node version available.

if you wanna use another version installed, list the available versions...
**nvm list**
And use:
**nvm use [version]**
To pick what you want.

Hope it helps!

**Comment by MMN3003** [comment]
**I fix it!!**
first uninstall node version that not work. in my case:
`nvm uninstall 16.13.0`
then run cmd/shell as administrator change my directory to :
`C:\`
then install node again
`nvm install 16.13.0`
and use it as current
`nvm use 16.13.0`
all done!
`nvm list`

**Comment by aditodkar** [comment]
> **I fix it!!** first uninstall node version that not work. in my case: `nvm uninstall 16.13.0` then run cmd/shell as administrator and change my directory to : `C:\` then install node again `nvm install 16.13.0` and use it as current `nvm use 16.13.0` all done! `nvm list`

Worked for me. Thanks a lot :)

**Comment by apzentral** [comment]
For me, in order for this to work. I need to use nvm version 1.1.7 instead of using 1.1.8.

**Comment by Onm-Galindo** [comment]
Using nvm version 1.1.7 instead of using 1.1.8 as well
